### DISCUSSION TITLE
Energy storage data

### DISCUSSION THREAD

**Discussion by guanxi99** [electricitymaps/electricitymaps-contrib#5108] [Ideas] [answered]
Hi team! Thx for that awesome app!
Would it be possible to determine and show the status of energy storage too?
I think first of all level of pumped hydro.
Rational: the app shows that critical interaction of renewables and non renewables in an excellent way, incl contribution to and consumption of pumped hydro. The missing link is how much is stored there at a given point of time or period on average.

**Comment by VIKTORVAV99** [comment]
Hi and thanks for the kind words!

We do already include energy storage in our values and graphs when the data is available. Sadly for most of Europe only Hydro Storage is available right now due to lack of data. However there is some good news, I actually reached out to ENTSO-E about this a few days ago and last I heard they are planning on adding battery storage to the transparency platform in Q4 2023 (it would still be up to TSOs to upload data though). That should significantly bring up our support for the battery storage system of the EU at least.

For other zones it would be on a zone to zone basis based on available data.

Is this what you had in mind when opening the discussion?

**Reply by guanxi99** [reply]
Hi Viktor, many thanks for your fas response. Not sure. To my understanding the app shows inflow and outlow to pumped hydro as negative and positive values, correct? But not yet the energy stored there at a given point of time or on average in a period of time..
If it does already, where can it be found?

**Reply by VIKTORVAV99** [reply]
That is true, we only show the net output/input of a storage mode and not the accumulated energy stored. While this is not something I have specifically discussed with the team some of the reasons for this is the lack of data and the magnitudes of energy it is about. For example ENTSO-E only have the weekly combined stored value for hydro and hydro storage and for week 2 in NO-NO2 the stored energy was 20997452 MWh. This is in comparison to battery storage installations that goes into thousands of MWh at most.

And while it could be useful for visibility and informational purposes it don't really affect anything else (at least in the short term). But I'll ask the team about it at our next meeting and see what they think.

**Reply by guanxi99** [reply]
Yes I agree that it does not impact CO2 emissions directly. Storage capacities and filling levels are only a key enabler for constant renewable energy supply

**Reply by tomber42** [reply]
Q4 2023 is now a long time ago.. But I can't see battery storage data. Is this available at ENTSO-E in the next time?

**Reply by VIKTORVAV99** [reply, marked as answer]
The data field is now available in ENTSO-E but no TSO is reporting any data yet. We have https://github.com/electricitymaps/electricitymaps-contrib/pull/7077 open and will add battery storage as soon as one TSO start reporting data so we have something to test it with.

**Reply by tomber42** [reply]
Hmmm.. are data now available?

**Reply by VIKTORVAV99** [reply]
As far as I have been able to tell none of the TSOs are reporting data for it yet. Nothing we can do about that I'm afraid but once they do we will add the data to our platform.
